includes/install-scripts/fresh-install.sh

```diff
@@ -12,13 +12,10 @@ rm -Rf /var/lib/mysql >>/dev/logs 2>&1
 ln -s /data/mysql /var/lib/mysql >>/dev/logs 2>&1
 source $START_SERVICES_SCRIPT
 
-/usr/bin/mysql -u root -e "CREATE DATABASE bluespice"
-/usr/bin/mysql -u root -e "CREATE USER 'bluespice'@'localhost' IDENTIFIED BY \"$rndpass\""
-/usr/bin/mysql -u root -e "GRANT ALL PRIVILEGES ON bluespice.* to 'bluespice'@'localhost'"
-/usr/bin/mysql -u root -e "FLUSH PRIVILEGES"
-sleep 5
-if [ -z $BS_LANG ]; then
-    BS_LANG="en"
+BS_DB_PASSWORD="ThisIsDBPassword"
```

Comment:
Looks like this can not be changed by using an EVN var. Be aware that now this value is being used instead of previous `BS_PASSWORD`. From my understanding, this will most likely break existing installations.

Reply:
No actually `BS_PASSWORD` is same as before (bluespice password) and `BS_DB_PASSWORD` was:

- earlier: was set as the `rndpass=$(date +%s | sha256sum | base64 | head -c 32 echo)` and was not configurable earlier
- now: is now set as `BS_DB_PASSWORD` as shown here and is not configurable now also